website/docs/components/form/masked-input/partials/code/how-to-use.md

```diff
@@ -130,6 +130,17 @@ Use the `@isRequired` and `@isOptional` arguments to add a visual indication tha
 </Hds::Form::MaskedInput::Field>
 ```
 
+#### Character count
+
+To support users meet content length constrains set `@minLength` and/or `@maxLength` to a `CharacterCount` contextual component.
+
+```handlebars
+<Hds::Form::MaskedInput::Field @value="036215df4996ca649928d8864b4df9e42cba0d" as |F|>
+  <F.Label>Terraform Cloud team token</F.Label>
+  <F.CharacterCount @minLength={{40}}/>
+</Hds::Form::MaskedInput::Field>
+```
+
```

Comment:
We should mention that we (HDS) don't provide "validation" for the character count, this is left to the consumer to implement, using the usual `<Error>` contextual component/messaging (and potentially provide an example of how to do it?).

Reply:
I tried addressing this in the last 3 commits, with a note that our implementation is very rudimentary (applications use more complex validation logic, a third-party such as `ember-validations` or a self-built solution as `cloud-ui` does).